src/session/session_dhandle.c

```diff
@@ -570,18 +569,18 @@ __wt_session_get_btree_ckpt(WT_SESSION_IMPL *session, const char *uri, const cha
      * Applications can use the internal reserved name "WiredTigerCheckpoint" to open the latest
      * checkpoint, but they are not allowed to directly open specific checkpoint versions, such as
      * "WiredTigerCheckpoint.6".
+     *
+     * There is no such restriction for the verification flow.
      */
     is_reserved_name = cval.len > strlen(WT_CHECKPOINT) && WT_PREFIX_MATCH(cval.str, WT_CHECKPOINT);
-    if (is_reserved_name)
+    if (is_reserved_name && !(flags & WT_BTREE_VERIFY))
```

Comment:
We tend to use this macro instead when a `flags` variable is accessible within the scope:
```suggestion
    if (is_reserved_name && !LF_ISSET(WT_BTREE_VERIFY))
```

Reply:
Done